sessions-core/src/main/java/com/adyen/checkout/sessions/core/SessionSetupInstallmentOptions.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by atef on 24/2/2023.
+ */
+
+package com.adyen.checkout.sessions.core
+
+import com.adyen.checkout.core.exception.ModelSerializationException
+import com.adyen.checkout.core.internal.data.model.JsonUtils
+import com.adyen.checkout.core.internal.data.model.ModelObject
+import com.adyen.checkout.core.internal.data.model.optIntList
+import com.adyen.checkout.core.internal.data.model.optStringList
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+@Parcelize
+data class SessionSetupInstallmentOptions(
+    val plans: List<String> = emptyList(),
+    val preselectedValue: Int? = 2,
```

Comment:
Why do we have `2` as a default value?

Reply:
we don't need it here actually, it was suppose to be inside example app